styles/components/alert.module.scss

```diff
@@ -0,0 +1,123 @@
+.popUpWrapper {
+    position: fixed;
+    top: 0;
+    left: 0;
+    background-color: rgba(68, 68, 68, 0.5);
+    margin: 0;
+    padding: 0;
+    opacity: 0;
+    z-index: 50;
+    width: 100%;
+    height: 100%;
+    visibility: hidden;
+    transform: scale(1.1);
+    transition: visibility 0s linear 0.25s, opacity 0.25s 0s, transform 0.25s;
+}
+
+.popUp {
+    box-sizing: border-box;
+    -webkit-box-sizing: border-box;
+    -moz-box-sizing: border-box;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    opacity: 1;
+    border-radius: 10px;
+    //z-index: 999;
+    overflow: hidden;
+    margin: 0;
+    width: 300px;
+    max-width: 100%;
+    height: 200px;
+    max-height: 100%;
+    transform: translate(-50%, -50%);
+    background-color: white;
+    color: #000;
+    font-weight: normal;
+}
+.popUpHeader {
+    background-color: rgb(252, 58, 58);
+    position: fixed;
+    width: 100%;
+}
+
+.popUpMessage {
+    height: 100%;
+    padding: 60px 10px 60px 10px;
+}
+
+.popUpFooter {
+    height: 50px;
+    position: fixed;
+    width: 100%;
+    bottom: 0px;
+}
+
+.popUpFooter button {
+    margin: 0;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    -ms-transform: translate(-50%, -50%);
+    transform: translate(-50%, -50%);
+    font-weight: normal;
+    color: black;
+}
+
+.popUpFooter button:hover, .popUp .popUpHeader button:hover {
+    background-color: rgb(129, 14, 14);
+    border-color: rgb(129, 14, 14);
+    color: white;
+
+}
+
+.popUp .popUpHeader button {
+    width: 10px;
+    float: right;
+    background-color: transparent;
+    min-width: 0px;
+    padding: 0.5rem;
+    width: 40px;
+    height: 40px;
+    margin-right: 10px;
+    margin-top: 5px;
+    transform: rotate(45deg);
+    font-size: 30px;
+    font-weight: bold;
+    color: black;
+}
+
+ .popUp>div{
+//     box-sizing: border-box;
+//     -webkit-box-sizing: border-box;
+//     -moz-box-sizing: border-box;
+//     margin: 0;
+//     padding: 5px;
+     min-height: 50px;
+//     font-size: 1em;
+//     font-weight: bold;
+//     text-align: center;
+//     color: #fff;
+//     background-color: transparent;
+//     margin: 0;
+//     padding: 0;
+//     border-style: none;
+//     width: 100%;
+//     padding: 5%;
+//     text-align: center;
+//     vertical-align: middle;
+//     border-width: 1px;
+//     border-color: #ccc;
+//     overflow: auto;
+ }
+
+.popUp button {
+    background-color: rgb(252, 58, 58);
+}
+
+.show {
+    opacity: 1;
+    visibility: visible;
+    transform: scale(1.0);
+    transition: visibility 0s linear 0s, opacity 0.25s 0s, transform 0.25s;
```

Comment:
Again, are all of these transitions valid?

Reply:
How do I check this?

Reply:
![Screenshot from 2021-03-29 15-51-15](https://user-images.githubusercontent.com/2736641/112839307-b6285700-90a6-11eb-92da-f0246e0a21ed.png)
Is this enough?